scania-digital-design-system/tegel
# feat(popover menu): design aligement

**Describe pull-request**  
Added fluid prop to allow for wider popover-menu. Aligned padding with design.

**Solving issue**  
Fixes: [CDEP-2442](https://tegel.atlassian.net/browse/CDEP-2442)

**How to test**  
1. Go to Popover Menu
2. Check that the component is aligned with design
3. Check the padding on the divider
4. Check the general padding
5. Test the fluid control
6. Make sure the fluid version is aligned with design


[CDEP-2442]: https://tegel.atlassian.net/browse/CDEP-2442?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
